inputfiles/browser.webidl.xml

```diff
@@ -3241,7 +3241,10 @@
         <method name="createHTMLDocument" type="Document">
           <param name="title" type="DOMString" />
         </method>
-        <method name="hasFeature" type="boolean" />
+        <method name="hasFeature" type="boolean">
```

Comment:
Please do not edit these files. these are imported and used as inputs. you need to edit https://github.com/Microsoft/TSJS-lib-generator/blob/master/inputfiles/overridingTypes.json instead